football_dictionaries/assignment_1.py

```diff
@@ -1,2 +1,16 @@
 def players_as_dictionaries(squads_list):
-    pass
+    squad_dict = []
+    for player in squads_list:
+        players_dict = {
+            "number": player[0],
+            "position": player[1],
+            "name": player[2],
+            "date_of_birth": player[3],
+            "caps": player[4],
+            "club": player[5],
+            "country": player[6],
+            "club_country": player[7],
+            "year": player[8],
+            }
+        squad_dict.append(players_dict)
+    return squad_dict        
```

Comment:
Great work on this assignment. I want to leave you with my example that I used just to show you a different approach. 


    labels = [
        'number',
        'position',
        'name',
        'date_of_birth',
        'caps',
        'club',
        'country',
        'club_country',
        'year'
    ]

    # will contain formatted list of dictionaries
    formatted_list = []

    # returns a collection of dictionaries with labels as keys
    # and values set to None
    for player_info in squads_list:
        player = dict.fromkeys(labels)
        formatted_list.append(player)

    # iterate over the squads list array
    for index, player in enumerate(squads_list):
        # iterate through the players in squads_list, and labels
        # updates the formatted list with individual player info
        for info, label in zip(player, labels):
            formatted_list[index].update({label: info})

    return formatted_list

I implemented this after considering the type of data structure we were working with. For me it seems like it helps to make the object we're working with much more flexible, in terms of making the right association with the keys, and values. For example, if we didn't know the indices of the data that we wanted to model, we wouldn't have to worry about specifying the index from our squads list. 

We would just add a label, and let the our algorithm figure out the rest for us.